src/widget/wbasiclabel.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <QLabel>
+
+/// Subclass of QLabel that allows the Up and Down arrow keys
+/// to be used like the Tab/Backtab keys.
+///
+/// See WLabel when you need to connect to a ControlObject.
+class WBasicLabel : public QLabel {
```

Comment:
@ronso0 I'm thinking whether it make sense to separate the skin widgets and the "basic" widgets into separate folders, e.g `src/widget/basic`. My gut feeling would be to create a more granular folder structure, though the current practice within Mixxx seems to favor bigger directories + prefixes.

As an aside, one could consider moving the skin widgets into `src/widget/skin` - though that's a lot of code churn for probably not a lot of benefit.

Reply:
Hmm, I'd rather stick to the `widgets` directory tbh.